Player/Player.gd

```diff
@@ -109,6 +110,12 @@ func _fly(delta: float) -> void:
 	if Input.is_action_pressed("move_left"):
 		direction -= aiming[0]
 	
+	if Input.is_action_pressed("slowmo"):
```

Comment:
is_action_just_pressed prevents double inputs from negating the input

Reply:
slow motion code should probably be in the _physics_process function as it's not movement code and would avoid duplication

Reply:
I don't think the keybind "slowmo" is set in this PR